packages/agent/src/app.ts

```diff
@@ -156,12 +157,38 @@ export class App {
           // @ts-expect-error - Deprecated message type
           case 'transmit':
           case 'agent:transmit:response':
-            this.addToHl7Queue(command);
+            if (this.config?.status !== 'active') {
+              const err = new Error('Agent.status is currently set to off');
+              this.log.error(normalizeErrorString(err));
+              this.addToWebSocketQueue({
+                type: 'agent:transmit:response',
```

Comment:
This should probably be `type: 'agent:error'`, no?

Reply:
Probably yes you're right